scania-digital-design-system/sdds
# feat(tabs): tab item

**Describe pull-request**  
This PR introduces 3 new components. folder-, inline- and navigation-tabs-item. These are three generic components that can be used to add a react router Link or NavLink component for the tabs.

**Solving issue**  
Fixes: -

**How to test**  
1. Go to storybook link below.
2. Check in Componets -> Tabs -> Folder/Inline/Navigation tabs
3. Change the "Tab type" to item and try the tabs out.
4. Check that the styling i still correct.

**Suggested test steps**
